test/functionality/_helper/dataflow/dataflow-builder-printer.ts

```diff
@@ -46,13 +46,15 @@ const EdgeTypeFnMap: Record<EdgeType, string | undefined> = {
 class DataflowBuilderPrinter {
 	private lines:           Lines = []
 	private graph:           DataflowGraph
+	private idMap:           DataflowMap
 	private rootIds:         Set<NodeId>
 	private coveredVertices: Set<NodeId> = new Set()
 	private coveredEdges:    Set<string> = new Set()
 
-	constructor(graph: DataflowGraph) {
+	constructor(graph: DataflowGraph, idMap: DataflowMap) {
```

Comment:
In general, whenever something besides flowr-core requires an idMap it should work without as well for debugging purposes.